app/cafe.py

```diff
@@ -0,0 +1,25 @@
+import datetime
+from app.errors import (NotVaccinatedError,
+                        NotWearingMaskError, OutdatedVaccineError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitors: dict) -> str:
+        flag = True
+        if "vaccine" not in visitors.keys():
+            flag = False
+            raise NotVaccinatedError
```

Comment:
The `NotVaccinatedError` should be raised with a descriptive message as per the checklist requirement. Consider using: `raise NotVaccinatedError('Visitor is not vaccinated.')`.